modules/sharethroughBidAdapter.js

```diff
@@ -81,30 +81,32 @@ export const sharethroughAdapterSpec = {
     }
 
     const imps = bidRequests.map(bidReq => {
-      let impression;
+      const impression = {};
+      impression.ext = {};
+
+      const gpid = utils.deepAccess(bidReq, 'ortb2Imp.ext.data.pbadslot');
+      if (gpid) {
+        impression.ext.gpid = gpid;
+      }
```

Comment:
we usually try to avoid sending empty objects in the query, we could do that instead
```suggestion
      const impression = {};

      const gpid = utils.deepAccess(bidReq, 'ortb2Imp.ext.data.pbadslot');
      if (gpid) {
        impression.ext = { gpid: gpid };
      }
```